semgrep/semgrep/cli.py

```diff
@@ -422,6 +414,17 @@ def cli() -> None:
         help=argparse.SUPPRESS,
         # help="Deprecated alias for --json + --debug",
     )
+    output.add_argument(
+        "--save-test-output-tar",
+        action="store_true",
+        help=argparse.SUPPRESS,
+        # help="Save --test output for use in semgrep-app registry",
+    )
+    output.add_argument(
+        "--synthesize-patterns",
```

Comment:
This is the old pattern recommendation functionality. It's also internal-only, and unused (?). Can we remove this from the Python interface at some point? cc @emjin